index.html

```diff
@@ -239,10 +239,164 @@ <h1 class="description-title">What is Real Dev Squad?</h1>
+                  companies will come across.
+                </p>
+              </div>
+              <button
+                type="button"
+                class="popup-trigger read-more"
+                data-popup-trigger="four"
+              >
+                Read More
+              </button>
+            </div>
+            <!-- Card 5 -->
+            <div class="card">
+              <h2 class="title-bullet-points">Engineering for evolution</h2>
+              <div>
+                <img
+                  src="img/cards/engineering-for-evolution.png"
+                  alt="real-dev"
+                />
+                <p>
+                  We are not a one-trick pony. We do projects in a lot of
+                  technologies, including standard web technologies like HTML,
+                  CSS, JavaScript, React, Ember, Vue, Node, Go, Rust, and more.
+                </p>
+                <p>
+                  We don’t limit ourselves to one thing. We focus on building
+                  amazing basic foundations and then adapting and learning
+                  whatever tech or tool we have to in order to engineer the best
+                  solutions to problems.
+                </p>
+                <p>
+                  We are engineers and problem solvers, not the kind of peoplee
+                  who will become outdated with time.
+                </p>
+              </div>
+              <button
+                type="button"
+                class="popup-trigger read-more"
+                data-popup-trigger="five"
+              >
+                Read More
+              </button>
+            </div>
+            <!-- Card 6 -->
+            <div class="card">
+              <h2 class="title-bullet-points">Soft Skills</h2>
+              <img src="img/cards/soft-skills.png" alt="real-dev" />
+              <div>
+                <p>
+                  Being successful in career is much more than just knowing how
+                  to push keys and click around. We ensure that you are the kind
+                  of person who is great at communitcation skills: Both Written
+                  and Spoken.
+                </p>
+                <p>
+                  We are ourselves a fun group, who believe in helping each
+                  other. You will have a good time!
+                </p>
+              </div>
+              <button
+                type="button"
+                class="popup-trigger read-more"
+                data-popup-trigger="six"
+              >
+                Read More
+              </button>
+            </div>
+            <div class="popup-modal shadow modal-card" data-popup-modal="one">
+              <i class="popup-modal__close"> X </i>
+              <h1 class="modal-title-bullet-points">Real Development</h1>
+              <div >
+                <img src="img/cards/real-devlopmennt.png" alt="real-dev" />
```

Comment:
Grammatical error in the name of `png` file

Reply:
fixed.